src/components/shared/basic/Input/BasicInput.tsx

```diff
@@ -157,14 +160,17 @@ const BasicInput = ({
       </div>
 
       <div style={{ width: '100%', height: '12px' }}>
-        {(errorMessage || (hint && focus)) && (
-          <Typography
-            sx={{ color: errorMessage ? Colors.error : Colors.secondary }}
-            variant={'label4'}
-          >
-            {errorMessage ?? hint}
-          </Typography>
-        )}
+        {
+          // eslint-disable-next-line
```

Comment:
Kindly provide an explanation for this disable linter

Reply:
Added disable linters here as we need OR condition and by using ?? the logic was not working

Reply:
is there no better solution?

Reply:
@jjeroch Nullish operator (??) return RHS only if LHS returns null or undefined but here first condition is boolean which either return true/false. Cases which is falling into false or empty, we have to disable it.